students/jjakub/session03/slicing_lab.py

```diff
@@ -15,8 +16,9 @@ def reverse(seq):
     return seq[::-1]
 
 def thirds(seq):
-    thrd = len(seq)//3
-    return seq[-thrd:] + seq[0:thrd + thrd]
+    thrd = int(round(len(seq)/3,0)*2)
+    return seq[thrd:] + seq[:thrd]
```

Comment:
This looks better (and probably correct), but the whole thing could have been simply as follows:
```
third = len(seq)//3
return seq[-third:] + seq[:third] + seq[third:-third]
```
Or maybe just `return seq[-third:] + seq[:-third]`. No need to change your code.

I'll leave this PR for your future commits for session 4. Thanks.